war/src/main/scss/components/_table.scss

```
@@ -13,13 +13,15 @@
   background-clip: padding-box;
   margin-bottom: var(--section-padding);
 
-  &::before {
-    content: "";
-    position: absolute;
-    inset: -4px -4px -2px;
-    border: var(--card-border-width) solid var(--table-border-color);
-    border-radius: inherit;
-    pointer-events: none;
+  @supports (not (-moz-appearance: none)) {
```

```suggestion
  // The '::before' pseudo-element is causing overflow issues in Firefox, so disable it for that browser
  @supports (not (-moz-appearance: none)) {
```

LGTM, would be worth adding a comment above the media query explaining why we have it.